frontend/src/login/LoginPage.jsx

```diff
@@ -88,7 +90,7 @@ class LoginComponent extends Component {
 
 const mapStateToProps = state => {
     return {
-        state: state.isLoggedIn
+        isValidated: state.isLoggedIn
```

Comment:
Should this be
isValidated: state.loginreducer.isloggedin?

Reply:
ooof it looks like I didn't commit properly T_T hold on fam

Reply:
oh wait thats the wrong file